spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/ConformanceExecution.scala

```diff
@@ -134,43 +136,48 @@ trait ConformanceExecution extends CommonJobExecution {
                                             fsUtils: FileSystemVersionUtils): Unit = {
     val cmdLineArgs: String = args.mkString(" ")
 
-    PerformanceMetricTools.addJobInfoToAtumMetadata(
-      "conform",
-      preparationResult.pathCfg.standardizationPath,
-      preparationResult.pathCfg.publishPath,
-      menasCredentials.username, cmdLineArgs
-    )
+    // TODO fix for s3 [ref issue #1416]
+//    PerformanceMetricTools.addJobInfoToAtumMetadata(
+//      "conform",
+//      preparationResult.pathCfg.standardizationPath,
+//      preparationResult.pathCfg.publishPath,
+//      menasCredentials.username, cmdLineArgs
+//    )
 
     val withPartCols = result
       .withColumnIfDoesNotExist(InfoDateColumn, to_date(lit(cmd.reportDate), ReportDateFormat))
       .withColumnIfDoesNotExist(InfoDateColumnString, lit(cmd.reportDate))
       .withColumnIfDoesNotExist(InfoVersionColumn, lit(preparationResult.reportVersion))
 
-    val recordCount = result.lastCheckpointRowCount match {
-      case None => withPartCols.count
-      case Some(p) => p
-    }
+    // TODO fix for s3 [ref issue #1416]
+    val recordCount = 100
+//    val recordCount = result.lastCheckpointRowCount match {
+//      case None => withPartCols.count
+//      case Some(p) => p
+//    }
     if (recordCount == 0) {
       handleEmptyOutput(SourcePhase.Conformance)
     }
 
     // ensure the whole path but version exists
-    fsUtils.createAllButLastSubDir(preparationResult.pathCfg.publishPath)
+    //fsUtils.createAllButLastSubDir(preparationResult.pathCfg.publishPath) // TODO fix for s3 [ref issue #1416]
 
     withPartCols.write.parquet(preparationResult.pathCfg.publishPath)
 
-    val publishDirSize = fsUtils.getDirectorySize(preparationResult.pathCfg.publishPath)
-    preparationResult.performance.finishMeasurement(publishDirSize, recordCount)
-    PerformanceMetricTools.addPerformanceMetricsToAtumMetadata(
-      spark,
-      "conform",
-      preparationResult.pathCfg.standardizationPath,
-      preparationResult.pathCfg.publishPath,
-      menasCredentials.username, cmdLineArgs
-    )
-
-    withPartCols.writeInfoFile(preparationResult.pathCfg.publishPath)
-    writePerformanceMetrics(preparationResult.performance, cmd)
+    // TODO fix for s3 [ref issue #1416]
+    //val publishDirSize = fsUtils.getDirectorySize(preparationResult.pathCfg.publishPath)
+    // preparationResult.performance.finishMeasurement(publishDirSize, recordCount)
+//    PerformanceMetricTools.addPerformanceMetricsToAtumMetadata(
+//      spark,
+//      "conform",
+//      preparationResult.pathCfg.standardizationPath,
+//      preparationResult.pathCfg.publishPath,
+//      menasCredentials.username, cmdLineArgs
+//    )
+
+    // TODO fix for s3 [ref issue #1416]
+    //withPartCols.writeInfoFile(preparationResult.pathCfg.publishPath)
+    //writePerformanceMetrics(preparationResult.performance, cmd)
 
     if (conformanceReader.isAutocleanStdFolderEnabled()) {
```

Comment:
I think this should also be commented out

Reply:
Thanks, done.